ccmlib/node.py

```diff
@@ -1660,6 +1660,10 @@ def _update_pid(self, process):
 
         try:
             with open(pidfile, 'r') as f:
+                if self.old_pids is None:
+                    self.old_pids = []
+                if self.pid is not None:
+                    self.old_pids += [self.pid]
```

Comment:
this is ok - yet I am not a pythonist - it may be preffered to use extend vs += 

@roydahan / @bentsi - whats your view

Reply:
why are you using old_pids - it incldued the current running pid - maybe all_pids is better (but I agree its not a real improvement)

Reply:
I used `old_pids` for backward compatibility so the code in dtest that relies on it could work with ccm without this change.  That said, `all_pids` could work too.
Regarding `+=`, besides it being used all over the place, `append(self.pid)` would be a better alternative for adding a single item (and `extend` for splicing a list / multiple elements)